src/frontend/src/common/components/library/data_subview/components/DownloadModal/index.tsx

```diff
@@ -173,43 +183,69 @@ const DownloadModal = ({
                   anchorEl,
                 }}
               >
-                <StyledSpan style={getBackgroundFastaColor()}>
-                  <CheckBoxWrapper>
-                    <CheckBoxInfo>
-                      {/* @ts-expect-error need to update checkbox state after upgrading sds */}
-                      <StyledCheckbox
-                        onChange={handleFastaClick}
-                        disabled={isFastaDisabled}
-                      />
-                    </CheckBoxInfo>
-                    <CheckBoxInfo>
-                      <DownloadType style={getBackgroundFastaColor()}>
-                        Consensus Genome{" "}
-                      </DownloadType>{" "}
-                      <span ref={tooltipRef}>(.fasta)</span>
-                      <DownloadTypeInfo>
-                        Download multiple consensus genomes in a single,
-                        concatenated file
-                      </DownloadTypeInfo>
-                    </CheckBoxInfo>
-                  </CheckBoxWrapper>
-                </StyledSpan>
+                <StyledFileTypeItem
+                  isDisabled={isFastaDisabled}
+                  isSelected={isFastaSelected}
+                >
```

Comment:
added `isDisabled` and `isSelected` as props to the styled component and moved the background color styling there.